openedx/features/edly/admin.py

```diff
@@ -0,0 +1,22 @@
+"""
+Django Admin pages for Edly.
+"""
+
+from django.contrib import admin
+
+from .models import EdlyOrganization, EdlySubOrganization
+
+
+class EdlyOrganizationAdmin(admin.ModelAdmin):
+    list_display = ['name', 'slug']
```

Comment:
Show edly sub organizations admin as an inline with their parent edly organization.
Also show created, updated timestamps

Reply:
Done. Kindly review the screenshot attached in the PR description.

Reply:
@asadmanzoor93 we still haven't added stacked inline `EdlySubOrganization` with zero extra. So that we can see all the related edly sub organizations in the edly organization detail page.

Reply:
Done